resolver.go

```diff
@@ -242,11 +258,42 @@ func (r *Resolver) SetText(opts *bind.TransactOpts, name string, value string) (
 
 // Text obtains the text associated with a name.
 func (r *Resolver) Text(name string) (string, error) {
-	nameHash, err := NameHash(r.domain)
```

Comment:
Please don't change variable names that are already set; it increases the size of the diff and makes it harder to understand.